workbench-core/base/src/aws/helpers/dynamoDB/ddbUtil.ts

```diff
@@ -12,7 +12,7 @@ export function buildDynamoDbKey(id: string, type: string): string {
   return `${type}#${id}`;
 }
 
-export function removeDynamoDbKeys(entry: { [key: string]: never }): { [key: string]: never } {
+export function removeDynamoDbKeys(entry: { [key: string]: unknown }): { [key: string]: unknown } {
```

Comment:
we don't read the values, so i think this should be `never`, right?

Reply:
I'm going to revert the changes I made to this code since I no longer need to use this method.